hc/accounts/tests/test_login.py

```diff
@@ -20,18 +20,23 @@ def test_it_sends_link(self):
         assert r.status_code == 302
 
         ### Assert that a user was created
+        self.test_user = User.objects.get()
+        self.assertEqual(self.test_user.email, "alice@example.org")
```

Comment:
maybe a better logic to test is assert that there is no user with a particular email, then create a user with that email, then assert that a user with that email now exixts and that the count of User objects has incremented by one.